sqlite/ext/comdb2/keywords.c

```diff
@@ -0,0 +1,182 @@
+/*
+   Copyright 2017 Bloomberg Finance L.P.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+#if (!defined(SQLITE_CORE) || defined(SQLITE_BUILDING_FOR_COMDB2)) &&          \
+    !defined(SQLITE_OMIT_VIRTUALTABLE)
+
+#if defined(SQLITE_BUILDING_FOR_COMDB2) && !defined(SQLITE_CORE)
+#define SQLITE_CORE 1
+#endif
+
+#include <assert.h>
```

Comment:
This file actually broke build with internal builddb in bbenv. Consider adding `#include <string.h>` for `memset`?